From: Dejan Latinovic <Dejan.Latinovic@imgtec.com>
Subject: ghc 7.8.2 FTBFS for mips/mipsel
Date: Fri, 13 Jun 2014 11:42:27 +0000

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=751479

A patch that adds mips and mipsel into NoSharedLibsPlatformList 
is attached.

After applying this patch,
I was able to successfully build ghc 7.8.2 for mips nad mipsel.

Index: ghc-7.8.3/mk/config.mk.in
===================================================================
--- ghc-7.8.3.orig/mk/config.mk.in	2014-07-18 22:46:32.190912601 +0200
+++ ghc-7.8.3/mk/config.mk.in	2014-07-18 22:46:44.622912143 +0200
@@ -99,7 +99,9 @@
 	x86_64-unknown-mingw32 \
 	i386-unknown-mingw32 \
 	sparc-sun-solaris2 \
-	sparc-unknown-linux
+	sparc-unknown-linux \
+	mips-unknown-linux \
+	mipsel-unknown-linux
 
 ifeq "$(SOLARIS_BROKEN_SHLD)" "YES"
 NoSharedLibsPlatformList += i386-unknown-solaris2
